.github/workflows/build_all.yml

```diff
@@ -17,31 +27,31 @@ jobs:
   build_publish_helm_charts:
     if: github.repository_owner == 'magma'
     env:
-      HELM_CHART_ARTIFACTORY_URL: "https://artifactory.magmacore.org:443/artifactory/"
-      HELM_CHART_MUSEUM_REPO: helm-test
-      HELM_CHART_MUSEUM_USERNAME: "${{ secrets.HELM_CHART_MUSEUM_USERNAME }}"
-      HELM_CHART_MUSEUM_TOKEN: "${{ secrets.HELM_CHART_MUSEUM_TOKEN }}"
+      HELM_CHART_ARTIFACTORY_URL: "https://linuxfoundation.jfrog.io/artifactory/"
+      HELM_CHART_MUSEUM_REPO: magma-helm-test
+      HELM_CHART_MUSEUM_USERNAME: "${{ secrets.LF_JFROG_USERNAME }}"
+      HELM_CHART_MUSEUM_TOKEN: "${{ secrets.LF_JFROG_PASSWORD }}"
       MAGMA_ROOT: "${{ github.workspace }}"
       EVENT_NAME: "${{ github.event_name }}"
       ISSUE_NUMBER: "${{ github.event.number }}"
     runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@7884fcad6b5d53d10323aee724dc68d8b9096a2e # pin@v2
+      - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # pin@v3.1.0
       # Version is github job run number when running on master
       # Or is branch name when on release branch
       - name: Set Helm chart version
         run: |
-          if [ "${GITHUB_REF##*/}" = "master" ] ;then
+          if [ "${GITHUB_REF}" = "refs/heads/master" ] ;then
             echo "VERSION=${GITHUB_RUN_NUMBER}" >> $GITHUB_ENV
-          elif [ "${EVENT_NAME}" = "pull_request" ]; then
```

Comment:
This is defined in line 25 `EVENT_NAME: "${{ github.event_name }}"` and should be removed there. I prefer the `"${{ github.event_name }}"` context instead of the env `"${GITHUB_EVENT_NAME}"`, but both should work.

Reply:
This PR is a backport to 1.8, i.e., goal is to have minimal impact and to diverge as less as possible from master. As this is also the case on master, we should not change this here.

Reply:
Fair point, I was not thinking about this aspect.